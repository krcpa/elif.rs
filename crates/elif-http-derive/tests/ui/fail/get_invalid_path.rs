//! Test that GET macro with invalid path argument fails

use elif_http_derive::get;

// This should fail - HTTP method macros expect string literals when args provided
#[get(invalid_path)]
pub async fn bad_handler() -> String {
    "Bad".to_string()
}

fn main() {}