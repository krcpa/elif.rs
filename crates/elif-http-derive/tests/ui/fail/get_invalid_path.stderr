error: expected string literal
 --> tests/ui/fail/get_invalid_path.rs:6:7
  |
6 | #[get(invalid_path)]
  |       ^^^^^^^^^^^^
